core/module/services.types.ts

```diff
@@ -55,4 +55,5 @@ export interface AuthService {
 export interface SubscriptionService {
   getSubscriptionLink(): Promise<{ url: string | null }>
   getActiveSubscription(): Promise<Record<string, string|number>>
+  getInvoices(): Promise<[Record<string, string | number>] | []>
```

Comment:
You mean `Array<Record<string, string | number >>`
Maybe can make an alias tyoe for this part `Record<string, string | number >` is this a reciept?